src/MinecraftSkins.php

```diff
@@ -30,18 +29,27 @@ public static function head($rawSkin, $scale = 1, $vertRot = 0, $horRot = 0, $he
 
         return $canvas;
     }
+	
+    public static function helm($rawSkin, $scale = 1, $vertRot = 0, $horRot = 0, $helmet = true) {
+        imagesavealpha($rawSkin, true);
+        //we create new **transparent** image
+        $canvas = self::createTransparent(self::HEAD_SIZE * $scale, self::HEAD_SIZE * $scale);
+
+        imagecopyresampled($canvas, $rawSkin, 0 * $scale, 0 * $scale, 40, 8
+                , self::HEAD_SIZE * $scale, self::HEAD_SIZE * $scale, 8, 8);
 
+        return $canvas;
+    }
     /**
      *
      * @param resource $rawSkin the raw skin data
      * @param int $scale size scale (default: 16 width * 32 height)
      * @param int $vertRot not implemented
      * @param int $horRot not implemented
-     * @param bool $helmet not implemented
      *
      * @return resource the rendered complete skin
      */
-    public static function skin($rawSkin, $scale = 1, $vertRot = 0, $horRot = 0, $helmet = true) {
+    public static function skin($rawSkin, $scale = 1, $vertRot = 0, $horRot = 0, $helmet = true, $secondskinlayer = true) {
```

Comment:
Please add the documentation for the last two parameters.

Reply:
Added some clarifications and comments. I'll add the documentation as soon as I have more free time.